foundations/javascript_basics/fundamentals-1.md

```diff
@@ -17,25 +17,11 @@ Look through these now and then use them to test yourself after doing the assign
 * What are assignment operators?
 * What is the "Unary +" Operator?
 
-### Variables
-
-You can think of variables as simply "storage containers" for data in your code. Until recently there was only one way to create a variable in JavaScript &mdash; the `var` statement. But in the newest JavaScript versions we have two more ways &mdash; `let` and `const`.
-
-1. [This variable tutorial](http://javascript.info/variables) will explain you everything you need to know! Be sure to do the __Tasks__ at the end. Information won't stick without practice!
-
-The above tutorial mentioned this, but it's important enough to note again: `let` and `const` are both relatively new ways to declare variables in JavaScript. In _many_ tutorials (and code) across the internet you're likely to encounter `var` statements. Don't let it bother you! There's nothing inherently wrong with `var`, and in most cases `var` and `let` behave the same way. But sometimes the behavior of `var` is _not_ what you would expect. Just stick to `let` (and `const`) for now. The precise differences between `var` and `let` will be explained later.
-
-### Numbers
-
-Numbers are the building blocks of programming logic!  In fact, it's hard to think of any useful programming task that doesn't involve at least a little basic math... so knowing how numbers work is obviously quite important.  Luckily, it's also fairly straightforward.
-
-1. [This W3Schools lesson](https://www.w3schools.com/js/js_numbers.asp) followed by [this one](https://www.w3schools.com/js/js_arithmetic.asp), are good introductions to what you can accomplish with numbers in JS.
-2. [This MDN article](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math) covers the same info from a slightly different point of view. There's much more that you can do with numbers, but this is all you need at the moment.
-3. Read through \(and code along with!\) [this article](http://javascript.info/operators) about operators in Javascript.  Don't forget to do the "Tasks" at the bottom of the page!  It will give you a pretty good idea of what you can accomplish with numbers (among other things!) in JavaScript.
+### How to Run JavaScript Code
 
-### Practice!
+All JavaScript we will be writing for the forseeable future will be ran via the browser. Later lessons in Foundations and the NodeJS path will show you how to run JavaScript outside of the browser environment. Keep in mind that due to differences in the NodeJS environment and the browser environment, for now you should always default to running your JavaScript in the browser unless otherwise specified.
```

Comment:
As a note, I added this bit because some people do try to use Node to run browser code via Code Runner and don't understand why they can't use `prompt()` or similar browser methods.

Reply:
I'm concerned that the jest section will be confusing for people after reading this.